keras/distribution/distribution_lib.py

```diff
@@ -410,12 +410,15 @@ class LayoutMap(collections.abc.MutableMapping):
     In the normal case, the key to query is usually the `variable.path`, which
     is the idenifier of the variable.
 
+    As shortcut, tuple or list of axis names are also allowed when inserting
+    as value, and will be converted to TensorLayout.
```

Comment:
Does this info even need to be user facing? Why would a user want to use TensorLayout directly?

Reply:
In the current case, the all the layout will use same mesh since that all the devices attached. However, in the multi-mesh case (for future), we would expect use to have different mesh specified for the corresponding layout, and weights/data could be redistributed to different mesh in a global view.